.github/workflows/build.yml

```diff
@@ -256,3 +256,48 @@ jobs:
           dev_tags: |
             docker.io/hashicorppreview/${{ env.repo }}:${{ env.dev_tag }}-ubi
             docker.io/hashicorppreview/${{ env.repo }}:${{ env.dev_tag }}-ubi-${{ github.sha }}
+
+  integration-tests:
+    name: Integration Tests (Consul ${{ matrix.server.version }})
```

Comment:
Why's the server version required in the test name? Shouldn't this be a constant?

Reply:
Good question! Consul Dataplane is compatible with multiple versions of the Consul server, starting with 1.14, to make upgrades easy — so we'll want to run the integration test suite against each of them to check we haven't broken anything on either side.